address_book/contacts/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+class Contact(models.Model):
```

Comment:
Can we use the built-in AbstractUser or BaseAbstractUser model here?

Reply:
No, `Abstract User` and `AbstractBaseUser` in Django are meant for managing user authentication, not for storing contact information like names, email, phone numbers, and addresses. In this project we don't need any authentication